pkg/bootstrap/os/tasks.go

```diff
@@ -454,3 +454,71 @@ func (u *UmountISO) Execute(runtime connector.Runtime) error {
 	}
 	return nil
 }
+
+type NodeConfigureNtpCheck struct {
+	common.KubePrepare
+}
+
+func (n *NodeConfigureNtpCheck) PreCheck(runtime connector.Runtime) (bool, error) {
+
+	if len(n.KubeConf.Cluster.System.NtpServers) == 0 && len(n.KubeConf.Cluster.System.Timezone) == 0 {
```

Comment:
if the user didn't set the `timezone`, this task will skip, does it equal your expectation?

Reply:
if the user didnot set timezone but set NtpServers, the task also will be executed.